.github/workflows/ci.yml

```diff
@@ -69,3 +69,8 @@ jobs:
 
     - name: Run Tests
       run: mix test
+
+    - name: ZAP Scan
```

Comment:
We should test against the local app... IMO

Because the code on Heroku is **before** the PR actually gets merged. That or we just set periodic pen tests to Heroku itself. Or the actual prod app. Like every day at midnight, instead on every commit

Reply:
Ok, request review on Slack when it's done 😛